data/json/items/magazine/762x25.json

```diff
@@ -0,0 +1,53 @@
+[
+  {
+    "id": "ppsh71drum",
+    "type": "MAGAZINE",
+    "name": "PPsh 71-round drum",
+    "description": "A high-capacity drum magazine for the PPSh-41.",
+    "weight": 1387, "//": "Provisional.",
+    "volume": 3,
+    "price": 7000,
+    "material": "steel",
+    "symbol": "#",
+    "color": "ltgray",
+    "ammo_type": "762x25",
+    "capacity": 71,
+    "reliability": 6,
+    "reload_time": 190,
+    "flags": [ "MAG_BULKY" ]
+  },
+  {
+    "id": "ppsh35mag",
+    "type": "MAGAZINE",
+    "name": "PPSh 35-round magazine",
+    "description": "A 35-round box magazine for the PPSh-41.",
+    "weight": 470,
+    "volume": 2,
+    "price": 2200,
+    "material": "steel",
+    "symbol": "#",
+    "color": "ltgray",
+    "ammo_type": "308",
+    "capacity": 30,
+    "reliability": 8,
+    "reload_time": 100,
+    "flags": [ "MAG_COMPACT" ]
+  },
+  {
+    "id": "ppsh71drum_imp",
+    "type": "MAGAZINE",
+    "name": "Improved PPsh 71-round drum",
+    "description": "A high-capacity drum magazine for the PPSh-41.  This one has thicker steel.",
+    "weight": 1387, "//": "Placeholder.",
+    "volume": 3,
+    "price": 10000,
+    "material": "steel",
+    "symbol": "#",
+    "color": "ltgray",
+    "ammo_type": "762x25",
+    "capacity": 71,
+    "reliability": 7,
+    "reload_time": 190,  "//": "Placeholder.",
```

Comment:
Why placeholder? I understand entirely for `weight` but `reload_time` is arbitrary?